### PR TITLE
8270929: Obsolete the FilterSpuriousWakeups flag in JDK 19

### PR DESCRIPTION
**Pull request by dholmes-ora**
Please review this small change to obsolete the `FilterSpuriousWakeups` flag that was deprecated in JDK 18.

There are no associated doc changes, nor test changes.

Testing: tier1-5 builds (in progress)

Thanks,
David

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8270929](https://bugs.openjdk.java.net/browse/JDK-8270929): Obsolete the FilterSpuriousWakeups flag in JDK 19

### Reviewers
 * [Harold Seigel](https://openjdk.java.net/census#hseigel) (@hseigel - **Reviewer**)
 * [Robbin Ehn](https://openjdk.java.net/census#rehn) (@robehn - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6871/head:pull/6871` \
`$ git checkout pull/6871`

Update a local copy of the PR: \
`$ git checkout pull/6871` \
`$ git pull https://git.openjdk.java.net/jdk pull/6871/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6871`

View PR using the GUI difftool: \
`$ git pr show -t 6871`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6871.diff">https://git.openjdk.java.net/jdk/pull/6871.diff</a>

</details>

### CODE REVIEW

**Review by hseigel — APPROVED** [review]
Changes look good and trivial.
Thanks, Harold

**Comment by dholmes-ora** [comment]
Thanks @hseigel ! It is a simple change but not quite trivial per the guidelines, so I'll wait for a second review.

**Review by robehn — APPROVED** [review]
Ship it!

**Comment by dholmes-ora** [comment]
Thanks @robehn !

/integrate

**Pull request closed without merging**
